backend/graminsta/post/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.0.8 on 2020-08-05 06:07
+
```

Comment:
Maybe add this file to .gitignore?

Reply:
I add this file according to PA's suggestion in [this PR](https://github.com/SPS-2020-Team-15/Graminsta/pull/1)
"Please generate migration files and submit them too, otherwise people will generate migration files on their own, which might make the migration diverge."
The core app in dev branch does have migration files, so I add this to post app :)

Reply:
Yes, more to this, for any of your changes related to `models.py`, you do need to try to create migration files and submit them. So all of you can remain the same database state. The migration files describe like:

```
Create model A with fields (a, b, c, d)
Update type of field `a` in model A to xxx
Delete field `b` of model A
...
```